uint8 MODE_CALIB       = 0
uint8 MODE_IDLE        = 1
uint8 MODE_CLOSE_LOOP  = 2
uint8 MODE_CLEAR_ERROR = 3
uint8 MODE_HOMING      = 4
uint8 MODE_SET_TARGET  = 5

uint8 device_id
uint8 mode
float32 value
